tap_pendo/__init__.py

```diff
@@ -39,6 +42,7 @@ class DependencyException(Exception):
 
 
 def validate_dependencies(selected_stream_ids):
+    # Validate and raise exceptions if sub-streams are selected but related parents not selected
```

Comment:
Validate and raise exceptions if sub-streams are selected but related parents are not selected